OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -1000,6 +1000,23 @@ protected override void OnFirstRun(Actor self)
 					foreach (var cell in rallyPoint)
 						QueueChild(new AttackMoveActivity(self, () => mobile.MoveTo(cell, 1, evaluateNearestMovableCell: true, targetLineColor: Color.OrangeRed)));
 			}
+
+			public override IEnumerable<Target> GetTargets(Actor self)
+			{
+				if (ChildActivity != null)
+					return ChildActivity.GetTargets(self);
```

Comment:
In the past we rewrote some recursive function calls. But trivial.

Reply:
this code is taken from here

https://github.com/OpenRA/OpenRA/blob/3111d06fcfc11130e435dabfe1d32ee860adbb75/OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs#L89-L98

Reply:
I'm not sure what you mean. All overwrites of GetTargets look like this